### PR TITLE
[DP-336] Adding worker for background processing

### PR DESCRIPTION
**Pull request by nate-taylor1**

### CODE REVIEW

File: charts/gitprime-app/v4.1.0/values.yaml [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+      - name: GP_TICKET_WEBHOOK_LISTENERS
+        value: 0
+      - name: GP_PR_WEBHOOK_LISTENERS
+        value: 0
+      - name: GP_PR_WEBHOOK_WORKERS_PER_LISTENER
+        value: 0
+      - name: GP_BP_LISTENERS
+        value: 1
+      - name: GP_BP_WEBHOOK_WORKERS_PER_LISTENER
+        value: 1
```
Comment (Alchemys) [inline charts/gitprime-app/v4.1.0/values.yaml]: I don't think we need any webhook listeners for Background processing

**Review by Alchemys — COMMENTED** [review]
Just one minor fix needed (don't need webhook listeners)

File: charts/gitprime-app/v4.1.0/values.yaml (hunk 1 above)
Comment (nate-taylor1) [reply charts/gitprime-app/v4.1.0/values.yaml]: Good catch.  I knew there would be at least one copy/paste error...

Comment (nate-taylor1) [reply charts/gitprime-app/v4.1.0/values.yaml]: Fixed.

File: charts/gitprime-app/v4.1.0/questions.yml [hunk 2]
```diff
@@ -0,0 +1,748 @@
...
+    description: The number of concurrent jobs for the BP workers.
+    default: 1
+
+  - variable: dataPipeline.bpWorker.bpThreadsPerListener
+    type: string
+    required: true
+    group: "Data Pipeline / BP Workers"
+    label: BP Worker Thread Count
+    description: The number of threads per job to use for the BP workers.
+    default: 1
```
Comment (dcumberland) [inline charts/gitprime-app/v4.1.0/questions.yml]: Does this configuration do anything?  I'm in favor of removing config options that do not have an impact on the application.  In this case, I don't think we use threads within BG processing.

**Review by dcumberland — CHANGES_REQUESTED** [review]
Can you remove the config option that's not needed?
For that matter, I think the same config option can and should be removed from tickets and PR processing.

In the future, we will move away from this format to the new helm repo, which doesn't use the copy/paste setup.  When working in this repo, I strongly encourage the first commit to be the copy/paste.  That way reviewers can exclude that commit when looking at the PR.

File: charts/gitprime-app/v4.1.0/questions.yml (hunk 2 above)
Comment (nate-taylor1) [reply charts/gitprime-app/v4.1.0/questions.yml]: Removed.

**Review by dcumberland — APPROVED** [review]
LGTM

**Pull request merged**